practice_flow_control_and_methods/last_letter_matcher.rb

```diff
@@ -1 +1,32 @@
-# Ask for two words. Say whether or not they end in the same letter.
\ No newline at end of file
+# Ask for two words. Say whether or not they end in the same letter.
+
+def get_words
+  puts 'word one:'
+  word_one = gets.chomp.downcase
+  puts 'word two:'
+  word_two = gets.chomp.downcase
+  return word_one, word_two
+end
+
+def identify_last_letter(word_one, word_two)
+  last_digit_one = word_one[-1]
+  last_digit_two = word_two[-1]
+  return last_digit_one, last_digit_two
+end
+
+def give_results(last_digit_one, last_digit_two)
+  if last_digit_one == last_digit_two
```

Comment:
you can change this line to

`if last_letters_match?(word_one, word_two)`

then you can change L11-15 to

```
def last_letters_match?(word_one, word_two)
  word_one[-1] == word_two[-1]
end
```

this would make your program a lot cleaner

Reply:
notice how i named this method

`last_letters_match?`

and it ends with a `?`. why do you think that is? when you see a method that ends in a question mark what can you expect the return value to be (by convention)?